lib/addons/amp/index.ts

```diff
@@ -116,23 +113,11 @@ export class AmpAddOn implements ClusterAddOn {
             }
             cfnWorkspace = new aps.CfnWorkspace(cluster.stack, 'MyAMPWorkspace', cfnWorkspaceProps);/* all optional props */ 
```

Comment:
Let's come up with a better ID than the hello world example of 'MyAMPWorkspace'. I am thinking something like workspaceName + "-amp-workspace".

Reply:
Fixed.